Example/Tests/WhatsNewTests.swift

```diff
@@ -17,37 +17,52 @@ class WhatsNewTests: XCTestCase {
         UserDefaults.standard.removePersistentDomain(forName: Bundle.main.bundleIdentifier!)
     }
     
+    func testShouldNotPresent_WhenCurrentVersionIsUndefined() {
+        XCTAssertFalse(WhatsNew.shouldPresent(with: .always, currentVersion: nil))
+    }
+    
+    
+    
```

Comment:
Can you please remove these spacing between the methods and only leave one empty line between each test? 👍

Reply:
yes